docs/hardware.rst

```diff
@@ -192,7 +226,12 @@ Offline Printer
 ^^^^^^^^^^^^^^^
 
 It is often useful to print submissions from the *Secure Viewing Station* for
-review and annotation.
+review and annotation. Printing eliminates most of the risks associated with
+malware, aside from malicious QR codes that a journalist may be tempted to scan.
+It also removes invisible metadata that may identify the source of a given
+document (with the exception of `printer tracking information <https://www.eff.org/issues/printers>`__
+included in a scanned document, or similar forms of metadata that may not be
+easily visible or recognizable).
```

Comment:
Great URL; maybe worth adding to the Journalist Guide -> Working with documents, as well, particularly this passage:

> Printing a file is often safer than exchanging it electronically, and it has the additional benefits of removing embedded metadata (except for printer codes, watermarks, or similar identifiers that may not be visible to the naked eye).

Reply:
Added link to malware section.